packages/clay-popover/src/index.tsx

```diff
@@ -42,6 +42,28 @@ const ALIGNMENTS_MAP = {
 	'top-right': ['br', 'tr'],
 } as const;
 
+type Point = typeof ALIGNMENTS_MAP[keyof typeof ALIGNMENTS_MAP];
+
+const BOTTOM_OFFSET = [0, 4] as const;
+const LEFT_OFFSET = [-4, 0] as const;
+const RIGHT_OFFSET = [4, 0] as const;
+const TOP_OFFSET = [0, -4] as const;
+
+const OFFSET_MAP = {
+	bctc: TOP_OFFSET,
+	blbr: RIGHT_OFFSET,
+	bltl: TOP_OFFSET,
+	brbl: LEFT_OFFSET,
+	brtr: TOP_OFFSET,
+	clcr: RIGHT_OFFSET,
+	crcl: LEFT_OFFSET,
+	tcbc: BOTTOM_OFFSET,
+	tlbl: BOTTOM_OFFSET,
+	tltr: RIGHT_OFFSET,
+	trbr: BOTTOM_OFFSET,
+	trtl: LEFT_OFFSET,
+};
```

Comment:
Hey @matuzalemsteles, why this is not moved to @clayui/shared? I think it can be reused in other domAlign usages around Clay

Reply:
Yeah, it's good to move there, not sure yet because we only have two places using this, so maybe it's not as necessary now, maybe when everything goes into `@clayui/core` package